Bundle/WidgetMapBundle/Resources/config/services.yml

```diff
@@ -5,9 +5,17 @@ services:
     victoire_widget_map.widget_data_warmer:
         class: Victoire\Bundle\WidgetMapBundle\Warmer\WidgetDataWarmer
         arguments:
-            - "@annotation_reader"
-            - "@victoire_view_reference.repository"
-            - ["\Victoire\Bundle\MediaBundle\Entity\Media"]
+            - @annotation_reader
+            - @victoire_view_reference.repository
+            - @victoire_widget.widget_helper
+            - WidgetSliderItem: WidgetSlider
```

Comment:
what if a new Entity needs to be added to this list ?
I think we should be able to populate this list so a simple solution is to use a parameter automatically configured by the bundle's configuration.